pkg/cluster/internal/create/actions/cluster/cluster.go

```diff
@@ -202,6 +203,12 @@ type TemplateParams struct {
 	DockerRegistries []map[string]interface{}
 }
 
+type Taint struct {
```

Comment:
do you think the format key1=value1:NoSchedule would be more readable?

this is the one used by kubectl https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/